scvi/model/_condscvi.py

```diff
@@ -3,6 +3,7 @@
 from typing import Optional, Union
 
 import numpy as np
+import scanpy
```

Comment:
we don't depend on scanpy directly, so we will need to do a lazy import of this inside the vamp prior function.

we also need to add leidenalg to our test dependencies.